src/auth/middleware/basic.js

```diff
@@ -1,20 +1,23 @@
 'use strict';
 
 const base64 = require('base-64');
-const User = require('../models/user-model.js');
+const User = require('../models/users.js');
 
 module.exports = async (req, res, next) => {
 
   if (!req.headers.authorization) { return _authError(); }
 
-  let basic = req.headers.authorization;
+  let basic = req.headers.authorization.split(' ').pop();
   let [user, pass] = base64.decode(basic).split(':');
 
   try {
     req.user = await User.authenticateBasic(user, pass)
     next();
   } catch (e) {
+    _authError()
+  }
+  function _authError () {
     res.status(403).send('Invalid Login');
+    
   }
```

Comment:
Good job getting that .env out of your github! Half of your tests are passing, would like to see those other tests passing.